.env.example

```diff
@@ -3,7 +3,6 @@ CHROMATIC_PROJECT_ID="Project:64762974a45b8bc5ca1705a2"
 # Note: the build script here should be defined in .storybook/package.json
 # this is used by the chromatic storybook addon to build the storybook on the fly
 CHROMATIC_BUILD_SCRIPT_NAME="build"
-CHROMATIC_ONLY_CHANGED=true
```

Comment:
If this is defaulted to true, it causes issues with the scoped runs, better to opt-into this setting